src/compressor/Compressor.cc

```diff
@@ -31,6 +31,7 @@ const char * Compressor::get_comp_alg_name(int a) {
 #ifdef HAVE_LZ4
   case COMP_ALG_LZ4: return "lz4";
 #endif
+  case COMP_ALG_BROTLI: return "brotli";
```

Comment:
should guard this case statement with `#ifdef HAVE_BROTLI`. as "brotli" could be disabled at compile-time.